src/main/resources/db/migration/V2_8__add_aktivitetskrav_varsel_publisert.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE AKTIVITETSKRAV_VARSEL ADD COLUMN published_at TIMESTAMPZ;
```

Comment:
Skulle vi prøvd å kalt dette feltet noe litt mer forståelig? At varselet er "published" tror jeg kan misforstås, for her er det snakk om at vi har plukket opp varselet etter fristen og sendt til veileder?
`oppgave_published_at` / `oppgave_sendt_date` / `oppgave_laget_date` / `oppgave_laget_at` ? Hva tenker du? 😊

Reply:
Dette feltet handler vel om publisering av varselet til `esyfovarsel`, og ikke oppgaven til `ispersonoppgave`? I så fall blir kanskje `varsel_published_at` et riktigere navn, men ettersom det allerede er et varsel gjennom tabell-navnet så kan det hende `published_at` fungerer 🤔 Evt `published_to_esyfovarsel_at` eller `published_to_kafka_at` 🤷🏼‍♂️

Reply:
Det handler vel om publisering av varselet til `esyfovarsel` ja - dvs om varselet er publisert/sendt til bruker. Sånn sett tenker jeg `published_at` fungerer fint (eller om man skal spesifisere til bruker på et vis). Tenker vi bør unngå å bruke appnavn/kafka o.l. i kolonnenavn :relaxed:

Reply:
Ja, virker som det er konsensus for å beholde published_at (og det samsvarer jo med det vi har i andre app'er).